### DISCUSSION TITLE
We are getting this error and the packages arent getting built properly

### DISCUSSION THREAD

**Discussion by Ro0412** [larics/icuas24_competition#25] [Q&A]
![Screenshot from 2024-01-06 20-50-22](https://github.com/larics/icuas23_competition/assets/123538724/965a6757-febc-4c0e-b84d-7c7bc67555cf)

**Comment by fpetric** [comment]
Hi, how are you building/installing stuff?

Note that the native installation of `uav_ros_simulation` is no longer available due to restricted permissions to certain repositories.
The preferred and easiest method is to simply build the Docker (in icuas24_competition repository) by running

`./docker_build.sh`
